diesel/src/macros/mod.rs

```diff
@@ -411,10 +429,11 @@ macro_rules! table_body {
         table_name = $name:ident,
         primary_key_ty = $primary_key_ty:ty,
         primary_key_expr = $primary_key_expr:expr,
-        columns = [$($column_name:ident -> $Type:tt; doc = [$($doc:expr)*],)*],
+        columns = [$($column_name:ident -> $Typey:tt; doc = [$($doc:expr)*]; real_name = $real_name:expr,)*],
```

Comment:
"Type Y"? What's the "y" for? (Personally, I'd of course change it to Typey McTypeface!)